Tests/FunctionalTests/Runner/Config.js

```diff
@@ -0,0 +1,16 @@
+var Config = {
+    RunnerPort: "9091",
+    TestAppAddress: "http://localhost:9099",
+    InstrumentationKey: "743e9ee1-1c7b-42c1-b41c-9cac8ea659e8",
```

Comment:
is it a real instrumentation key? if not maybe change it to `imafake-instr-umen-tati-onkey1111111` :)

Reply:
Actually not used anymore! Removed